.github/workflows/ci-tests.yml

```
@@ -7,20 +10,30 @@ on:
       run: npm ci
-      
+
+    - name: Build extension (dev mode)
+      run: npm run webpack
+
     - name: Run tests
-      uses: GabrielBB/xvfb-action@v1
-      with:
-        run: npm run test
\ No newline at end of file
+      run: xvfb-run npm run test
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)